config/samples/nnf_v1alpha1_nnflustremgt.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfLustreMGT
+metadata:
+  labels:
+    app.kubernetes.io/name: nnf-sos
+    app.kubernetes.io/managed-by: kustomize
+  name: nnflustremgt-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
Do you plan on filling this out? If not, then I'd say just remove this file.

Reply:
No, we don't use these for anything. The ones that are filled out are all stale. I'll just delete it